test/screens/alerts/alert_test.exs

```diff
@@ -110,6 +113,13 @@ defmodule Screens.Alerts.AlertTest do
 
   describe "fetch_from_cache/2" do
     setup do
+      default_stops_mod = Application.get_env(:screens, :alerts_cache_filter_stops_mod)
+      Application.put_env(:screens, :alerts_cache_filter_stops_mod, Stop.Mock)
```

Comment:
This may cause inconsistent test failures due to this test module being `async: true`. Is there any problem with always using the mock version of the module in `test.exs`? I'm not sure if we'd ever want it to be the "real" one since that would imply making real network requests.